README.md

```diff
@@ -5,6 +5,10 @@
 
 Probabilistic programming with NumPy powered by [JAX](https://github.com/google/jax) for autograd and JIT compilation to GPU/CPU.
 
+[Docs](https://readthedocs.org/projects/numpyro/badge/?version=0.1.0) | [Examples](https://pyro.ai/numpyro/) | [Forum](https://forum.pyro.ai/)
```

Comment:
Can we use `stable` badge instead: `https://readthedocs.org/projects/numpyro/badge/?version=stable`? [version=0.1.0](https://readthedocs.org/projects/numpyro/badge/?version=0.1.0) seems do not work.